config/routes.rb

```diff
@@ -57,8 +57,10 @@
     patch '/orders/:id', to: "dashboard#update"
     patch '/merchants/:id', to: "merchants#update"
     get '/merchants/:merchant_id', to: "dashboard#show"
+    get '/merchants/:merchant_id/orders/:order_id', to: "orders#show"
     get '/merchants/:merchant_id/items', to: "items#index"
     patch '/merchants/:merchant_id/items/:item_id', to: "items#update"
+    patch '/merchants/:merchant_id/orders/:order_id/items/:item_id', to: "items#update"
     delete '/merchants/:merchant_id/items/:item_id', to: "items#destroy"
     get '/merchants/:merchant_id/items/new', to: "items#new"
```

Comment:
Add two routes for the admin order show page and the item update for a certain order